pages/windows/color.md

```diff
@@ -0,0 +1,15 @@
+# color
+
+> Set the console foreground and background colors.
+
+- Set the console colors to the default values:
+
+`color`
+
+- Set the console foreground and background to a specific color:
+
+`color {{color_hex}}`
+
+- List available color values and detailed information:
```

Comment:
This should be above `color {{hex_code}}` as the users will need to know what values are present to be able to set a color.

Reply:
That's a good point. I'll move it above.